### PR TITLE
 corrected all errors

### PR DESCRIPTION
**Pull request by ranshu1601**
their were errors in main.py file and function.py file which is now resolved. 
[ gssoc participant ]

### CODE REVIEW

File: helpers/functions.py [hunk 1]
```diff
@@ -412,11 +425,14 @@ def duration(
...
     total_payment = n * frequency
     coupon_payment = coupon_rate / frequency * face_value
-    payment = [coupon_payment] * (total_payment - 1) + [coupon_payment + face_value]
+    payment = [coupon_payment] * \
+        (total_payment - 1) + [coupon_payment + face_value]
     data["period"] = pd.DataFrame(np.arange(1, total_payment + 1))
     data["payment"] = pd.DataFrame(payment)
-    data["dcoupon"] = data["payment"] / ((1 + rate / frequency) ** data["period"])
-    data["pv"] = data["dcoupon"] / frequency * data["period"] / data["dcoupon"].sum()
+    data["dcoupon"] = data["payment"] / \
```
Comment (ighoshsubho) [inline helpers/functions.py:432]: What does forward slash mean?

File: helpers/functions.py [hunk 2]
```diff
@@ -412,11 +425,14 @@ def duration(
...
+        ((1 + rate / frequency) ** data["period"])
+    data["pv"] = data["dcoupon"] / frequency * \
```
Comment (ighoshsubho) [inline helpers/functions.py:434]: Here as well..

File: helpers/functions.py [hunk 3]
```diff
@@ -936,7 +960,8 @@ def calculate_mortgage_interest(
     annual_interest_rate = percent_to_decimal(annual_interest_rate)
     loan_amount = mortgage_amount * percent_to_decimal(100 - mortgage_deposit)
     power = (1 + annual_interest_rate) ** loan_term
-    mortgage_annual_payment = loan_amount * (annual_interest_rate * power) / (power - 1)
+    mortgage_annual_payment = loan_amount * \
```
Comment (ighoshsubho) [inline helpers/functions.py:963]: Here too..

**Comment by ranshu1601** [comment]
I haven't added these slashes , but i will just rectify it and push code again .  

> There are more forward slashes and just fix them, I will merge them accordingly!

File: helpers/functions.py [hunk 4]
```diff
@@ -962,7 +987,8 @@ def calculate_fha_mortgage_interest(
     loan_term_months = loan_term * 12
     monthly_interest_rate = percent_to_decimal(annual_interest_rate) / 12
     power = (1 + monthly_interest_rate) ** loan_term_months
-    monthly_payment = mortgage_amount * (monthly_interest_rate * power) / (power - 1)
+    monthly_payment = mortgage_amount * \
```
[the remark this answers is not in the document]
Comment (ranshu1601) [reply helpers/functions.py:990]: image
so actually when I am saving the file , a part of code moves to the next line on it's own .

**Comment by ighoshsubho** [comment]
Ok fine I got it, thanks @ranshu1601, your PR got merged 🎉, Keep contributing like this in GSSOC'23!

GIF

**Pull request merged**